### PR TITLE
[LIB-377] Models

### PR DESCRIPTION
**Pull request by jakubbilko**

### CODE REVIEW

File: src/models/apikey.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import stampit from 'stampit';
+import {Meta, Model} from './base';
+
+const ApiKeyMeta = Meta({
+  name: 'api_key',
```
Comment (dancio) [inline src/models/apikey.js]: use camel case :)

File: src/models/codeboxtrace.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import stampit from 'stampit';
+import {Meta, Model} from './base';
+
+const CodeBoxTraceMeta = Meta({
+  name: 'codeboxtrace',
+  pluralName: 'codeboxtraces',
+  endpoints: {
+    'detail': {
+      'methods': ['delete', 'patch', 'put', 'get'],
+      'path': '/v1/instances/{instance}/codeboxes/{codebox_id}/traces/{trace_id}/'
```
Comment (dancio) [inline src/models/codeboxtrace.js]: camelcase? `codeboxId`, `traceId` ?

Comment (jakubbilko) [inline src/models/codeboxtrace.js]: f233b3b2bcdc3632f0a99e89061586de0f996077

**Comment by dancio** [comment]
lgtm

**Pull request merged**